tests/test_pylint.py

```diff
@@ -1,61 +0,0 @@
-#! /usr/bin/env python
```

Comment:
Let's get also rid of the following:

- [unused requirement in CI setup](https://github.com/zeek/zeekscript/blob/407ed31a8191fd7e561a49ddbe1423868c2131bb/.github/workflows/build_wheels.yml#L46)
- [reference in README](https://github.com/zeek/zeekscript/blob/407ed31a8191fd7e561a49ddbe1423868c2131bb/.github/workflows/build_wheels.yml#L46)

Reply:
Nice, thank you — pushed.